cli/network.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/alibaba/pouch/apis/types"
+
+	"github.com/spf13/cobra"
+)
+
+// NetworkCommand is used to implement 'network' command.
+type NetworkCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkCommand command.
+func (n *NetworkCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "network [command]",
+		Short: "Manage pouch networks",
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	c.AddCommand(n, &NetworkCreateCommand{})
+}
+
+// Run is the entry of NetworkCommand command.
+func (n *NetworkCommand) Run(args []string) {}
+
+// NetworkCreateCommand is used to implement 'network create' command.
+type NetworkCreateCommand struct {
+	baseCommand
+
+	name       string
+	driver     string
+	gateway    string
+	ipRange    string
+	ipamDriver string
+	ipamOpts   []string
+	subnet     []string
+	options    []string
+	labels     []string
+}
+
+// Init initializes NetworkCreateCommand command.
+func (n *NetworkCreateCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "create <NAME>",
+		Short: "Create a pouch network",
```

Comment:
Please add Long and Example for new initialized Command, like https://github.com/alibaba/pouch/blob/master/cli/create.go#L28-L33.
They could help a lot in doc generation.

Reply:
ok